onnxruntime/python/onnxruntime_validation.py

```diff
@@ -65,6 +65,10 @@ def validate_build_package_info():
         has_ortmodule = True
     except ImportError:
         has_ortmodule = False
+    except EnvironmentError:
+        # ORTModule is but not ready to run yet
```

Comment:
Don't fully understand this sentence (just the comment "ORTModule is but not ready to run yet")

Reply:
ops, it was meant to be: "ORTModule is present, but not ready yet"